neo/Settings.py

```diff
@@ -147,9 +147,23 @@ def setup_testnet(self):
         """ Load settings from the testnet JSON config file """
         self.setup(FILENAME_SETTINGS_TESTNET)
 
-    def setup_privnet(self):
-        """ Load settings from the privnet JSON config file """
+    def setup_privnet(self, host=None):
+        """
+        Load settings from the privnet JSON config file
+
+        Args:
+            host (string): can be either an IP which will be used for seeds with standard ports and RPC
+                           or in the format of `IP:port`, which will be used as the only seed
```

Comment:
How would you use this `IP:port` format? It looks like colons aren't supported below, but I could be missing something...

Reply:
Oh i left that comment in. The problem I faced with port was that you'd need 2 ports: at least a p2p and a rpc server port. Didn't have an idea about how to do this properly, so i thought let's start simple and improve later. If you have any ideas i'd be happy to hear them! ;)